README.md

```diff
@@ -67,7 +68,6 @@ The following steps apply when running CHT locally in development mode and when
 
 #### CHT Development Environment
 1. Set up a local CHT instance using [these instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/).
```

Comment:
Following [these instructions](https://docs.communityhealthtoolkit.org/apps/tutorials/local-setup/) I get to a port conflict with interoperability. More detail on issue [comment](https://github.com/medic/interoperability/issues/24#issuecomment-1468453919)